Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/getsentry/sentry-cocoa",
         "state": {
           "branch": null,
-          "revision": "0ef2893efb9b27d471fa3e4e0ccd199ec7a5e726",
-          "version": "7.2.1"
+          "revision": "0a915b93ff3abee1a9752448e47808334d306980",
```

Comment:
Can we revert this change? We should try and avoid committing version changes that aren't directly related to the ticket, since other projects are using this library and we can't be sure it won't have unintended consequences.